input/pagecontent/general-requirements.md

```diff
@@ -411,7 +264,94 @@ AU Core MedicationRequest|MedicationRequest.code.coding|[Australian Medication](
-- AU Core Requesters **SHALL** be capable of meaningfully processing a Medication resource with `Medication.code.coding` with both.
-- Systems **MAY** support populating and processing other terminologies but this is not a requirement of AU Core.
+- AU Core Responders **SHALL** correctly populate `Medication.code.coding` with either a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both, if a coded value is known.
+- AU Core Requesters **SHALL** accept a Medication resource without error if `Medication.code.coding` is present and containing any valid value. A valid value may be text, or may be a code from [Australian Medication](https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1) or [PBS Item Codes](https://build.fhir.org/ig/hl7au/au-fhir-base//ValueSet-pbs-item.html), or both, or some other code.
+
+Systems **MAY** populate and accept other code systems but this is not a requirement of AU Core.
+
+### Missing Data
+
+There are situations when information for a particular data element is missing and the source system does not know reason for the absence of data. 
+
+#### Missing Must Support and Optional Data
+
+If the source system does not know the value for an *Optional* element (minimum cardinality = 0), including elements labelled *Must Support*, the data element **SHALL** be omitted from the resource.  
+
+#### Missing Must Support and Mandatory Data
+
+If the data element is a *Mandatory* element (minimum cardinality is > 0), the element **SHALL** be present *even if* the source system does not know the value or the reason the value is absent. The core specification provides guidance for what to do in this situation, which is summarised below.
+
+1.  For *non-coded* data elements where the applicable AU Core profile does not mandate a sub-element
+    - use the [DataAbsentReason extension](http://hl7.org/fhir/R4/extension-data-absent-reason.html) 
+    - use the code `unknown` _The value is expected to exist but is not known_
+  
+    Example: Patient resource where the patient's birthDate is not available.
+    ~~~
+    {
+      "resourceType" : "Patient",
+           ...
+           "_birthDate" : [
+               {
+                "extension" : [
+                  {
+                    "url" : "http://hl7.org/fhir/StructureDefinition/data-absent-reason",
+                    "valueCode" : "unknown"
+                  }
+                ]
+              }
+           ],
+          ...
+         }
+    ~~~
+      
+2. For *non-coded* data elements where the applicable AU Core profile mandates a sub-element, e.g. in AU Core PractitionerRole, the sub-element `Practitioner.name.family` is mandatory, then the resource must contain the sub-element otherwise the resource will not be conformant.
```

Comment:
AU Core PractitionerRole should be AU Core Practitioner.

Reply:
fixed